media-sound/ladish/ladish-1.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit python-single-r1 waf-utils
+
+MY_P="${P}-g36c489e4"
+DESCRIPTION="LADI Session Handler - a session management system for JACK applications"
+HOMEPAGE="https://ladish.org"
+SRC_URI="https://dl.ladish.org/ladish/${MY_P}.tar.bz2"
+S="${WORKDIR}/${MY_P}"
+
+# common/klist.h is linked list code borrowed from linux kernel, and thus is GPL2 only
+# otherwise it is GPL-2+ code
+LICENSE="GPL-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+# media-sound/a2jmidid[dbus] is optional but recommended runtime-only dependency
+IUSE="debug +lash +a2jmidid"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/expat
+	media-sound/jack2[dbus]
+	sys-apps/dbus
+	virtual/jack
+	${PYTHON_DEPS}
+	a2jmidid? ( media-sound/a2jmidid[dbus] )
+	lash? ( !!media-sound/lash )
```

Comment:
`!!` blockers are the worst UX. It means it cannot even be autoresolved by portage and media-sound/lash must be removed prior to emerging ladish, because it must not be installed during building ladish (?). which I doubt.

If it is just installed files colliding with lash, then a regular `!` blocker is enough.